Features/Context/SubContext/CommonActions.php

```diff
@@ -314,6 +314,32 @@ private function goToContentWithPath($path)
     }
 
     /**
+     * @Then I should see a :button action bar button
+     *
+     * @param  string   $buttonText     Text of the element in the action bar
+     */
+    public function iShouldSeeActionBarButton($buttonText)
+    {
+        $element = $this->getElementByText($buttonText, '.ez-actionbar-container .ez-action', '.action-label');
+        if (!$element) {
+            throw new \Exception("Action bar button '$buttonText' not found");
```

Comment:
File seems to be using `EzAssertion` already, should it keep doing so, for consistency?
```
EzAssertion::assertElementFound($buttonText, $element, null, 'Action bar button');
```